src/integrationTest/java/uk/gov/hmcts/darts/common/service/AutomatedTaskServiceTest.java

```diff
@@ -0,0 +1,136 @@
+package uk.gov.hmcts.darts.common.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.config.CronTask;
+import org.springframework.scheduling.config.FixedDelayTask;
+import org.springframework.scheduling.config.FixedRateTask;
+import org.springframework.scheduling.config.ScheduledTask;
+import org.springframework.scheduling.config.ScheduledTaskHolder;
+import org.springframework.scheduling.config.Task;
+import org.springframework.scheduling.config.TriggerTask;
+import uk.gov.hmcts.darts.common.entity.AutomatedTaskEntity;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.task.AutomatedTask;
+import uk.gov.hmcts.darts.common.task.ProcessDailyListAutomatedTask;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Slf4j
+class AutomatedTaskServiceTest extends IntegrationBase {
+    @Autowired
+    private AutomatedTaskService automatedTaskService;
+
+    @Autowired
+    private ScheduledTaskHolder scheduledTaskHolder;
+
+    @Test
+    void givenConfiguredTasksCancelTaskAndAddTaskBackWithRandomCronExpression() throws InterruptedException {
+        Thread.sleep(10_000);
+
+        AutomatedTask automatedTask = new ProcessDailyListAutomatedTask();
+        Optional<AutomatedTaskEntity> originalAutomatedTaskEntity =
+            automatedTaskService.getAutomatedTaskEntityByTaskName(automatedTask.getTaskName());
+        log.info("----- Original task {} cron expression {}", automatedTask.getTaskName(),
+                 originalAutomatedTaskEntity.get().getCronExpression());
+
+        Set<ScheduledTask> scheduledTasks = scheduledTaskHolder.getScheduledTasks();
+        displayTasks(scheduledTasks);
+
+        log.info("----- About to cancel task {}", automatedTask.getTaskName());
+        boolean taskCancelled = automatedTaskService.cancelAutomatedTask(automatedTask.getTaskName(), false);
+        assertTrue(taskCancelled);
+
+        Thread.sleep(21_000);
+        log.info("----- Tasks after sleep");
+        scheduledTasks = scheduledTaskHolder.getScheduledTasks();
+        displayTasks(scheduledTasks);
+        automatedTaskService.updateAutomatedTaskCronExpression(automatedTask.getTaskName(), getRandomCronExpression());
```

Comment:
Does the cron expression need to be random? Can we just specify a known cron value instead?